### ISSUE TITLE
swiftmailer mail delivery to address xezpeleta+eb@tknika.eus after initial setup

### ISSUE THREAD

**Issue by metzgerd** [elkarbackup/elkarbackup#411]
After installation all emails are send to xezpeleta+eb@tknika.eus which is configured as delivery_addresses in swiftmailer

```
./console debug:config swiftmailer

Current configuration for extension with alias "swiftmailer"
============================================================

swiftmailer:
    default_mailer: default
    mailers:
        default:
            transport: sendmail
            host: 'mail.example.com:465'
            username: mail@example.com
            password: 
            delivery_addresses:
                - xezpeleta+eb@tknika.eus
            url: null
            port: null
            timeout: 30
            source_ip: null
            local_domain: null
            encryption: null
            auth_mode: null
            logging: true
            delivery_whitelist: {  }
```

`May 22 13:24:07 elkarbackup postfix/smtp[745]: B432F28C6B: to=<xezpeleta+eb@tknika.eus>, relay=aspmx.l.google.com[173.194.76.27]:25, delay=1.5, delays=0.22/0.05/0.67/0.57, dsn=2.0.0, status=sent (250 2.0.0 OK DMARC:Quarantine 1590153847 o132si7635190wma.43 - gsmtp)
`
Workaround: Remove entry in /usr/local/elkarbackup/app/config/config_dev.yml

Do not know why I'm running in dev mode ...

**Comment by elacunza** [comment]
How did you install elkarbackup? Debian packages? Docker images?

**Comment by metzgerd** [comment]
I've installed via apt on Ubuntu 20.04. 
Since 6 weeks I'm using the Docker Image. Did not have the issue with Docker.

**Comment by elacunza** [comment]
Fixed in #430
